Code/Tools/ProjectManager/Source/ProjectUtils.cpp

```diff
@@ -119,36 +228,63 @@ namespace O3DE::ProjectManager
                     return false;
                 }
 
-                QGuiApplication::setOverrideCursor(QCursor(Qt::WaitCursor));
-                copyResult = CopyProject(origPath, newPath);
-                QGuiApplication::restoreOverrideCursor();
-
+                copyResult = CopyProject(origPath, newPath, parent);
             }
 
             return copyResult;
         }
 
-        bool CopyProject(const QString& origPath, const QString& newPath)
+        bool CopyProject(const QString& origPath, const QString& newPath, QWidget* parent)
         {
             // Disallow copying from or into subdirectory
             if (IsDirectoryDescedent(origPath, newPath) || IsDirectoryDescedent(newPath, origPath))
             {
                 return false;
             }
 
-            if (!CopyDirectory(origPath, newPath))
+            QStringList filesToCopy;
+            qint64 totalSizeInBytes = 0;
+
+            QProgressDialog* progressDialog = new QProgressDialog(parent);
+            progressDialog->setAutoClose(true);
+            progressDialog->setValue(0);
+            progressDialog->setRange(0, 1000);
+            progressDialog->setModal(true);
+            progressDialog->setWindowTitle("Copying project ...");
```

Comment:
might eventually want to wrap this text in tr()

Reply:
Added